ui/components/app/edit-gas-fee-popover/edit-gas-item/edit-gas-item.test.js

```diff
@@ -38,7 +41,12 @@ const MOCK_FEE_ESTIMATE = {
   estimatedBaseFee: '50',
 };
 
-const renderComponent = (props) => {
+const DAPP_SUGGESTED_ESTIMATE = {
+  maxFeePerGas: '0x59682f10',
```

Comment:
Would it be better to represent these as decimals so that it's easier to compare with the assertions below? I've started using this trick in tests:

``` javascript
import { BN } from 'ethereumjs-util';

// ...

`0x${new BN(someAmountInWei).toString(16)}`
```

Reply:
hex value in above scenario looks better to me